script/Ciel/Day 14/14_01A_CIEL14A_4.txt

```diff
@@ -666,32 +720,35 @@ And yet, there isn't a single cause of death on Arcueid.
 [sha:46e440a646dd4b68891723e2111bb899d79b4142]{
 -- Page 141, Offset 38022.
 -- あれはこの自然界の上にいるだけで完璧な生命だと、前に誰かが言っていた。
-There was someone who once told me that she was a perfect lifeform only while she stands on top of this realm of nature.
+// They did? I can't find any other mentions of 自然界 or 完璧な生命 in Ciel route other than D14
+Someone once told me that so long as she exists within nature, she is a perfect lifeform.
 }
 [sha:f25c283e497b659b62ce387f6a832d53cfe01aa4]{
 -- Page 141, Offset 38023.
 -- 自然の延長である真祖は、この世界という地盤からいくらでも活力を引き上げられる。
-A True Ancestor, who is an extension of nature itself, can pull as much energy as they want from it.
+True Ancestors, who are extensions of nature itself, can draw as much energy as they want from the planet.
 }
 [sha:1b85e11e44501f04bc949ae1ea6f10fdf414e993]{
 -- Page 141, Offset 38024.
 -- だから死なない。寿命がない。限界というものがない。
-That's why she can't die. She doesn't have a lifespan. There's no such thing as a limit for her.
+That's why she can't die.
+ She doesn't have a lifespan.
+ There is no limit to her existence.
 }
 [sha:70f42007195476f6cc3d4c55369dbf8f10183e5c]{
 -- Page 142, Offset 38025.
 -- 「――――」
 "――――"
 }
-[sha:03fba19c19ab441a43df39ed93ac1063df02e956]{
+[offset:38026]{
 -- Page 142, Offset 38026.
 -- つまり。
-In other words―――
+But in other words―――
```

Comment:
So feels so left out.

>So in other words―――

Was thinking maybe this was an alternative too, I think it flows a bit better.
>Which means―――